dashboard/app/views/scripts/show.html.haml

```diff
@@ -32,13 +32,15 @@
 
   .row#stats
     .span12#rightside
-      - if @script.trophies
-        = render partial: 'shared/user_stats', locals: { user: current_user, width: '100%', async: true }
-      - else
-        .user-stats-block
-          :javascript
-            dashboard.progress.renderCourseProgress(#{@script.summarize.to_json});
-          = render partial: 'shared/user_stats_key'
+      .user-stats-block
+        :javascript
+          dashboard.progress.renderCourseProgress(#{@script.summarize.to_json});
```

Comment:
Responded on the other PR.  For publicly-cached scripts we don't know if the user is logged in or not, so we need to hit the API regardless.

The API returns `{}` immediately if there is no `current_user`.